stages/release.md

```diff
@@ -6,12 +6,11 @@
 ### __Stage description:__  
 Insert your description of the stage here  
```

Comment:
@kdlyit I think we also need to add a description about this stage, specially because other teams would need this stage on their logbook. Here is a small suggestion:

"The release stage is between the **Test** and **Deploy** stages, here is where we can implement the CI/CD in our pipeline, but first it needs to how we are going to test and deploy the code, in this case we have decided the release will have three stages:
* **Release to QA**: Here is where the code will be released to QA environment, so it can be properly tested (manually or automated tests) by both developers and QA Engineers, assuming there is no issues/bugs found, the code will then be released to Staging environment.
* **Release to Staging (if no issues in QA)**: The code will be related to Staging environment and all automated testes will be triggered, assuming the code changes didn't break tests, the release will proceed to the next stage.
* **Release to Production (if no issues in Staging)**: At this point the code has been properly tested and should not cause any unexpected issues in production, therefore it should be safe to deploy to production.

In case an issue is reported in one of those stages, a rollback strategy will need to be done, so it doesn't cause issues to any user, a simple strategy is to revert the latests code changes and do a new release with the fix later.
"

Reply:
@kdlyit Once you agree with the description, let's add that to the PR, and then we can finally merge it.